### ISSUE TITLE
Feature request: Batch document creation and reading

### ISSUE THREAD

**Issue by DRK3** [decentralized-identity/confidential-storage#138]
Hi (again!),

Another useful feature that many databases have is the ability to create multiple documents at once (or read multiple documents at once). Having this in EDV would be useful as well.

**Comment by OR13** [comment]
we might want upsert many, we can also use the indexes to do "batch read".

I would classify this as "batch upsert"

**Comment by DRK3** [comment]
Yes, +1 to having upsert functionality built in to the batching functionality as well. [I needed to add that functionality to the TrustBloc EDV server as an optional extension](https://github.com/trustbloc/edv/pull/166) in order to allow for batching functionality in the [EDV REST provider](https://github.com/hyperledger/aries-framework-go/blob/master/pkg/storage/edv/restprovider.go) so we could improve performance.

**Comment by DRK3** [comment]
(Renamed the issue to be a little more general.. specifically, it would be beneficial to be able to batch any operation within a vault, including upsert and delete)

**Comment by OR13** [comment]
@DRK3 sounds like you made a specific endpoint for batching operations? feel free to open a PR adding that endpoint to the spec :)

**Comment by OR13** [comment]
You may also want to comment on this PR, which attempts to define operations: https://github.com/decentralized-identity/confidential-storage/pull/140#pullrequestreview-542889609

perhaps batch operations belong here?

**Comment by dmitrizagidulin** [comment]
@OR13 - agreed, yeah, I actually link to this issue here from that PR 140 (as a placeholder in the spec to discuss batch requests)

**Comment by dmitrizagidulin** [comment]
Oh, whoops, I only linked to the delete vault issue. Will add a link to this batch document issue shortly.

**Comment by DRK3** [comment]
@OR13 @dmitrizagidulin I see that there's a new repo for the EDV spec at https://github.com/decentralized-identity/edv-spec. Can this issue be moved over?

**Comment by OR13** [comment]
yes, I have added the label, to move it will be reviewing and moving all labeled issues in our next edv topic call for the wg
